ridley/src/System/Metrics/Prometheus/Ridley/Types.hs

```diff
@@ -177,10 +176,13 @@ makeLenses ''RidleyCtx
 
 instance Katip Ridley where
   getLogEnv = Ridley $ lift (lift getLogEnv)
+  localLogEnv = localLogEnv
 
 instance KatipContext Ridley where
   getKatipContext   = return mempty
+  localKatipContext = localKatipContext
```

Comment:
I think these changes mean that you need a lower bound on `katip`.  You can either find when these were introduced and use that or just go with the more conservative `katip >= 0.6 && < 0.7`.

Reply:
unfortunately there was no lower bound that didn't cause a TON of dependency mismatches =(